articles/migrations/index.md

```diff
@@ -25,6 +25,42 @@ If you need help with the migration, create a ticket in our [Support Center](${e
 
 Current migrations are listed below, newest first. For migrations that have already been enabled see [Past Migrations](#past-migrations).
 
+### Whitelisting IP Address Ranges Australia
+
+| Severity | Grace Period Start | Mandatory Opt-In|
+| --- | --- | --- |
+| Low | 2017-08-22 |  2017-09-30 |
+
+Auth0 is expanding into new global regions, and traffic originating from these regions will have new IP addresses. If you are whitelisting IP addresses, you will need to add the new addresses to your firewall rules.
```

Comment:
> Auth0 is expanding into new global regions
Auth0 is updating its cloud environments, and traffic from these regions will originate from new IP addresses. If you are ...